releasenotes/notes/diagnose-info-log-ae1b2adf9a60ae58.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    The ``agent diagnose`` and ``agent flare`` commands no longer reate error-level log messages when the diagnostics fail.
```

Comment:
```suggestion
    The ``agent diagnose`` and ``agent flare`` commands no longer create error-level log messages when the diagnostics fail.
```

Reply:
This is one of those personal failings I've never been able to shake: awful typos in whatever textual thing I edited last before making a PR :(